geode-redis/src/main/java/org/apache/geode/redis/internal/executor/string/SetOptions.java

```diff
@@ -53,6 +63,31 @@ public boolean isKeepTTL() {
     return keepTTL;
   }
 
+  @Override
+  public int getDSFID() {
+    return REDIS_SET_OPTIONS_ID;
+  }
+
+  @Override
+  public void toData(DataOutput out, SerializationContext context) throws IOException {
+    DataSerializer.writeObject(exists, out);
```

Comment:
since "exists" is always an Enum of type Exists, its serialization can be optimized by directly calling DataSerializer.writeEnum and readEnum

Reply:
fixed